src/aposteriori/data_prep/cli.py

```diff
@@ -113,11 +113,12 @@
 @click.option(
     "-ae",
     "--atom_encoder",
-    type=click.Choice(["CNO", "CNOCB", "CNOCBCA", "CNOCBCAQ", "CNOCBCAP"]),
+    type=click.Choice(["CNO", "CNOCB", "CNOCACB", "CNOCACBQ", "CNOCACBP"]),
```

Comment:
For backwards compatibility, can we add both?

type=click.Choice(["CNO", "CNOCB", "CNOCACB", "CNOCACBQ", "CNOCACBP", "CNOCBCA", "CNOCBCAQ", "CNOCBCAP" ]),